python/cudf/cudf/_lib/cpp/types.pxd

```diff
@@ -102,3 +103,5 @@ cdef extern from "cudf/types.hpp" namespace "cudf" nogil:
         HIGHER "cudf::interpolation::HIGHER"
         MIDPOINT "cudf::interpolation::MIDPOINT"
         NEAREST "cudf::interpolation::NEAREST"
+
+    ctypedef unsigned long long int128
```

Comment:
why unsigned?

Reply:
This is a bit of a hack we had to do for cython to support `int128`: https://stackoverflow.com/a/27583541

Reply:
I think we can avoid this with https://github.com/rapidsai/cudf/pull/9533/files#r784233626.